flink-runtime/src/main/java/org/apache/flink/runtime/state/DefaultOperatorStateBackend.java

```diff
@@ -196,6 +196,20 @@ public void removeOperatorState(String name) {
 		}
 	}
 
+	public void deleteBroadCastState(String name) {
```

Comment:
I think you added these methods twice, once under `deleteBroadcastState` and once under `deleteBroadCastState`.

Reply:
Argh...will remove the second one ;)